client/src/test/java/org/apache/rocketmq/client/consumer/rebalance/AllocateMessageQueueConsitentHashTest.java

```diff
@@ -92,9 +92,9 @@ public void testAllocate2() {
 
     @Test
     public void testRun100RandomCase() {
-        for (int i = 0; i < 100; i++) {
-            int consumerSize = new Random().nextInt(200) + 1;//1-200
-            int queueSize = new Random().nextInt(100) + 1;//1-100
+        for (int i = 0; i < 10; i++) {
+            int consumerSize = new Random().nextInt(20) + 1;//1-20
```

Comment:
@shroman we can talk about these problems in dev mail. How to unit test, integration-test, performance test and stress test, and what's the main goal when we do it :-)